### DISCUSSION TITLE
Public documentation functionality (A default welcome page/collection + login button in public pages)

### DISCUSSION THREAD

**Discussion by PovilasID** [outline/outline#4157] [Ideas]
Hello,

**Use case:** 

I want to use Outline as internal **documentation** tool and I do not need for all user to be logged, but for them to have that option. My use case is similar to what https://app.getoutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/ I am looking for quality of life improvements to make it more seamless.

**Changes/Updates:**

1. **Login button in public pages.** I will direct my users to a public page/collection and I want them to to have an option to transition into editing that page. Ideal would be to to have a button "Login" in a page and after rlogin in redirecting back to last visited page or at least to their home page.
2. **Custom default home page**. When guest going to outlines domain instead of seeing a login page he would see a "default welcome/homepage collection/page" Additional option next to collection permissions ("Set this as default public homepage" etc) would be nice. Outline's own docs url looks...long `https://app.getoutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/` I would prefer sharing something more human friendly like `https://docs.mydomain.com/` This is similar to #2223 

Context: I am selfhosting my instance do to regulation. 

P.S. I saw a couple off issues there... people who are looking for support for selfhosting are being disrespectful because they project opinions onto developers and get... petulant... I am sorry you have to deal with that. I really appreciate this project hope you keep growing and maturing it 👍

**Comment by tommoor** [comment]
Hi, what you're describing matches with our plans – which is to enable a public view of teams which would function much like a public help center. Currently work is underway to allow multiple teams under a single login, once that's complete we'll be investigating how to make a team public.

**Reply by PovilasID** [reply]
Is there an expected timeline?
Months? Weeks? "Planned for 23Q2" etc?

**Reply by tommoor** [reply]
Realistically it's looking like Q1 '23

**Comment by marcbria** [comment]
Just to understand... would be possible to do something like this for visitors with outline?
https://docs.getoutline.com/s/guide

Would be really useful to share a collection with the world... while you keep working in private stuff.

Thanks for the great project,
m.

**Reply by tommoor** [reply]
It's possible with the cloud-hosted version yep, that page is just a shared Outline document itself. If you want to setup something similar email hello@getoutline.com

**Reply by PovilasID** [reply]
Is there a feature comparison table between selfhosted  and cloud version?

**Reply by marcbria** [reply]
And maybe a roadmap to know when new features will be transfer to the community version?

I'm sorry but this is a personal/spare-time project and I can't afford the paid version.

I'm trying to find out if it's worth the jump from hackmd, because in functionality you both are very close and for me, this would be an important element to tip the balance.

**Comment by EduNetEurope** [comment]
Archbee has "spaces" (= collections of collections), which can have their own URL and be shared publicly or be private.
Within these spaces, you can decide for each folder = collection and document indiviudally whether it should be shared publicly or not.

We can't wait for Outline to have something like spaces. Sharing collections is nice but it would not allow us to share a wiki publicly.
It would be great if it was possible to share collections and to have folders within these collections.

The wording should be improved as well. It is very confusing that _publish_ = 'share internally' and _share_ = 'publish to internet'.

**Reply by tommoor** [reply]
> to have folders within these collections.

We don't plan on having a separate concept for folders, because documents can be nested an empty document acts the same as a folder. Our own help guide is just a shared document, for example:

https://docs.getoutline.com/s/guide

**Reply by EduNetEurope** [reply]
Makes sense.
Are there any plans to add the ability to publish collections publicly with different urls so that we can share several wikis online?
We need separate wikis for each project and it would be amazing to have them all in one workspace since it would allow us to copy documents between the different wikis. However, each wiki should be able to have its own logo, each own url, its own colour scheme, etc.
